packages/devtools_app/lib/src/result.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+// run `flutter pub run build_runner build --delete-conflicting-outputs` to re-generate
+part 'result.freezed.dart';
+
+@freezed
+abstract class Result<T> with _$Result<T> {
```

Comment:
Document what this class is and why it is useful?
If it is only used for the Provider screen and riverpod screens move it under that directory.